src/features/import/components/Importer/validation/AddedTagsTracker.tsx

```diff
@@ -0,0 +1,62 @@
+import { Box } from '@mui/system';
+import { Typography } from '@mui/material';
+
+import { Msg } from 'core/i18n';
+import TagChip from 'features/tags/components/TagManager/components/TagChip';
+import useTags from 'features/tags/hooks/useTags';
+import { ZetkinTag } from 'utils/types/zetkin';
+
+import messageIds from 'features/import/l10n/messageIds';
+
+interface AddedTagsTrackerProps {
+  createdTags: { byTag: { [key: number]: number }; total: number };
+  orgId: number;
+}
+const AddedTagsTracker: React.FunctionComponent<AddedTagsTrackerProps> = ({
+  createdTags,
+  orgId,
+}) => {
+  const { data } = useTags(orgId);
+  const tags = data || [];
+
+  const addedTags = Object.keys(createdTags.byTag).reduce(
+    (acc: ZetkinTag[], id) => {
+      const tag = tags.find((tag) => tag.id === parseInt(id));
+      if (tag) {
+        return acc.concat(tag);
+      }
+      return acc;
+    },
+    []
+  );
```

Comment:
I'm thinking this logic could be moved into a hook, and maybe even that the user of this component calls the hook and gives this component the data it needs?